apps/aecore/src/aec_peers.erl

```diff
@@ -864,7 +864,7 @@ on_peer_accepted(PeerAddr, PeerInfo, Pid, State0) ->
                                     "from ~s to ~s; rejecting connection",
                                     [ppp(PeerId), format_address(OldPeer),
                                      format_address({PeerAddr, Port})]),
-                    {error, trusted_address_changed}
+                    {{error, trusted_address_changed}, State}
```

Comment:
Not sure of impact of this hence whether release note line needed.

Reply:
@lucafavatella this fixes a bug in a corner case where a known trusted peer would change its IP. It would crash without this fix.